libass/ass_directwrite.c

```diff
@@ -352,6 +352,19 @@ static bool check_postscript(void *data)
 }
 
 /*
+ * Lazily return index of font. It requires the FontFace to be present, which is expensive to initialize.
+ */
+static unsigned get_font_index(void *data)
+{
+    FontPrivate *priv = (FontPrivate *)data;
+
+    if (!init_font_private_face(priv))
+        return false;
```

Comment:
Maybe that should be 0 for clarity. Since the function doesn't actually return a bool, but an unsigned index.